### PR TITLE
Add Files Action to Search

### PR DESCRIPTION
**Pull request by alperozturk96**
**Behaviour:**
https://github.com/nextcloud/android/assets/67455295/d09e263c-4311-4391-8f99-57ed8e46e42f

This feature needed for [this](https://github.com/nextcloud/android/pull/12092) bugfix

- [x] Tests written, or not not needed

### CODE REVIEW

File: app/src/main/res/layout/unified_search_item.xml [hunk 1]
```diff
@@ -107,5 +107,19 @@
...
+    <ImageButton
+        android:id="@+id/more"
+        android:visibility="gone"
+        android:background="@color/transparent"
+        app:srcCompat="@drawable/ic_dots_vertical"
+        app:tint="@color/standard_grey"
+        android:layout_gravity="center"
+        android:layout_width="50dp"
+        android:layout_height="50dp"
+        android:contentDescription="open file actions"
```
Comment (ZetaTom) [inline app/src/main/res/layout/unified_search_item.xml:120]: Please replace the hardcoded text with an `R.string` based one. We already have a similar button in `list_item.xml`, called `overflow_menu`, which could be used as a template.

**Review by ZetaTom — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/unified_search_item.xml (hunk 1 above)
Comment (alperozturk96) [reply app/src/main/res/layout/unified_search_item.xml]: Fixed

**Review by JonasMayerDev — APPROVED** [review]

**Comment by tobiasKaminsky** [comment]
For me it does not work.
When clicking on "three dots" of a search item, it opens root folder in background.
This looks different in your video, and seems also be working for Jonas.

[Screen_recording_20231130_072915.webm](https://github.com/nextcloud/android/assets/5836855/36a798ab-987c-46e4-8ae9-db75750d86b7)

**Comment by alperozturk96** [comment]
> For me it does not work. When clicking on "three dots" of a search item, it opens root folder in background. This looks different in your video, and seems also be working for Jonas.
> 
> EDIT: after reviewing above video, it is the same. So I think this is a bit wrong, as still the search view should be shown. Imagine that an user accidently clicks on "three dots" for wrong file and then needs to search again.

It's same for selecting file. If user tap wrong file needs to search again. So the question has to be asked here are we going to change all search functionality with better selection and showing files action or do we leave the selection process as it is.

Given the current structure of the project, this solution seems appropriate.

**Review by JonasMayerDev — APPROVED** [review]
Works for me :)

**Pull request merged**